### PR TITLE
Attempted to fix MuAnsiHandler on non-windows platforms

### PR DESCRIPTION
**Pull request by matthewfcarlson**
On non-windows platforms, it fails without certain windows based loads. This commit attempted o re-order the loading stages to prevent that from happening.

### CODE REVIEW

**Comment by matthewfcarlson** [comment]
I was focused on these two tests just simply importing the library and seeing if any exception were thrown. I beefed up the tests, so let me know what you think

File: MuPythonLibrary/MuAnsiHandler.py [hunk 1]
```diff
@@ -47,6 +47,73 @@ def winapi_test():
 else:
     from ctypes import byref, Structure
 
+    # inspired by https://github.com/tartley/colorama/
```
Comment (corthon) [inline MuPythonLibrary/MuAnsiHandler.py:54]: Now that there's this much logic in the try-catch-else, I feel like we should have some comments explaining what each path represents.

File: MuPythonLibrary/MuAnsiHandler_test.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+
+class MuAnsiHandlerTest(unittest.TestCase):
+
+    # we are mainly looking for exception to be thrown
+
+    def test_formatter(self):
```
Comment (corthon) [inline MuPythonLibrary/MuAnsiHandler_test.py]: For good test-driven design/unit testing, each of these cases should tell a story and be as small as possible.

E.g. test_formatter() -> test_formatter_should_add_color_codes_for_critical()

Comment (corthon) [reply MuPythonLibrary/MuAnsiHandler_test.py]: And test_formatter_should_contain_some_output_when_a_record_is_passed

**Review by corthon — CHANGES_REQUESTED** [review]

File: MuPythonLibrary/MuAnsiHandler.py (hunk 1 above)
Comment (matthewfcarlson) [reply MuPythonLibrary/MuAnsiHandler.py:54]: Agreed

File: MuPythonLibrary/MuAnsiHandler_test.py (hunk 2 above)
Comment (matthewfcarlson) [reply MuPythonLibrary/MuAnsiHandler_test.py]: Agreed

**Review by corthon — APPROVED** [review]

**Pull request merged**